sample-apps/segmentation_spleen_scribbles/main.py

```diff
@@ -33,10 +33,10 @@ def init_infers(self):
             "Spleen_Segmentation": SegmentationWithWriteLogits(
                 self.final_model, load_from_mmar(self.mmar, self.model_dir)
             ),
-            "BIFSeg+CRF": SpleenBIFSegCRF(),
```

Comment:
Can we define/describe all these methods in README.md for the app.

Reply:
Added description for these methods into README